src/main/java/com/exadel/discount/model/http/RefreshResponse.java

```diff
@@ -0,0 +1,11 @@
+package com.exadel.discount.model.http;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public class RefreshResponse {
+    private final String accessToken;
+    private final String refreshToken;
```

Comment:
refresh token is generated only on login. No regeneration

Reply:
This is done.